utils/src/main/kotlin/DeclaredLicenseProcessor.kt

```diff
@@ -88,13 +90,15 @@ object DeclaredLicenseProcessor {
     /**
      * Try to map all [declaredLicenses] to a (compound) [SpdxExpression] by taking both hard-coded mappings and
      * [declaredLicenseMapping] into account. As a special case, a license may be mapped to [SpdxConstants.NONE] to mark
-     * it as something that is not a license, like a copyright that was accidentally entered as a license. Return a
+     * it as something that is not a license, like a copyright that was accidentally entered as a license. Multiple
+     * declared licenses are reduced to a [SpdxCompoundExpression] using the specified [operator]. Return a
      * [ProcessedDeclaredLicense] which contains the final [SpdxExpression] (or null if none of the declared licenses
      * could be mapped), and the mapped and unmapped licenses respectively.
      */
     fun process(
         declaredLicenses: Set<String>,
-        declaredLicenseMapping: Map<String, SpdxExpression> = emptyMap()
+        declaredLicenseMapping: Map<String, SpdxExpression> = emptyMap(),
+        operator: SpdxOperator = SpdxOperator.AND
```

Comment:
Would it make sense to allow setting this via `PackageCurationData`? E.g. for use cases where `AND` was intended (Maven).

Reply:
I don't think so. For such cases we could already curate the declared license with a string that concatenates the individual licenses with an "AND" string.